src/api/java/com/minecolonies/api/colony/IColonyTagCapability.java

```diff
@@ -97,7 +104,7 @@
         /**
          * The set of all close colonies. Only relevant in non dynamic claiming.
          */
-        private final Set<Integer> colonies = new HashSet<>();
+        private Set<Integer> colonies = new HashSet<>();
```

Comment:
Question:
Why remove the final here and make it so that we *set* the colonies (close colonies, should probably rename this field)
I think it'd be better to use an add (to list) method and keep it final. Then you could also add close colonies later on (say a new colony is placed down later) without getting, modifying, and setting, or elsewise

Reply:
it is so that we can override the client's chunk data with new data on receiving a message